spec/support/tagging_shared_examples.rb

```diff
@@ -113,7 +119,12 @@
 
       let(:endpoint) { "untag" }
       it_behaves_like "bad_tags"
-      it_behaves_like "good_tags"
```

Comment:
Just wanted to point out this removal, it doesn't really make sense that this line was here because the `good_tags` shared example doesn't test the `untag` endpoint, so it was a duplication. Editing the `good_tags` example to use the `endpoint` variable breaks specs and so should be addressed separately.